tests/init_test.py

```diff
@@ -0,0 +1,11 @@
+import unittest
+import requests
+
+class Tester(unittest.TestCase):
+	def test_home(self):
+		response = requests.get("http://127.0.0.1:5000")
```

Comment:
Is this meant to run on the Vagrant VM or on the developer machine? If its the latter, 127.0.0.1 is probably not the right address to connect to. If its the former, there should be instructions somewhere to tell oyur peers how to run test.

Reply:
Its not a file that we created as part of this PR.
In fact this file is approved as part of issue #7.
I just changed the name of the folder containing it.